requirements.pip

```diff
@@ -1,3 +1,5 @@
 # Our dependencies are all specified in setup.py.
+# If we're using this requirements file, we almost certainly want Django extras
+# as well.
 
--e .
\ No newline at end of file
+-e .[django]
```

Comment:
It's the syntax for specifying that a package should be installed with a particular set of extras: https://pythonhosted.org/setuptools/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies